danieldupriest/reimbursinator
# Mega bug fix

This update makes boolean fields display as a select box with a yes/no option and fixes the non-unique id problem we had with input boxes.

To test:
Edit or create a report, change the flight info response, and click save. You should get an alert showing that international now has the value you assigned it. Close the modal popup, refresh the page, and edit the same report. The change should persist.

Right click the top of the form and select inspect/inspect element. Now find the the div with the class "modal fade show". Right click on it and select expand recursively on the context menu. When you scroll down the id attribute of inputs, forms, and collapse divs should be prefixed with section-number- where number is that element's corresponding section id.
